dapp/src/components/dashboard/pages/Dashboard/index.jsx

```diff
@@ -48,54 +82,59 @@ const Dashboard = (props) => {
-        homeTokenAddress={homeTokenAddress}
-        foreignTokenAddress={community && community.foreignTokenAddress}
-      />
-      {
-        (community && community.foreignTokenAddress) && (
-          <div className='content__bridge'>
-            <div className='grid-x align-justify'>
-              <h3 className='content__bridge__title'>Bridge <FontAwesome style={{ fontSize: '60%' }} data-tip data-for='bridge' name='info-circle' /></h3>
-              <ReactTooltip className='tooltip__content' id='bridge' place='bottom' effect='solid'>
-                <div>Use the bridge to move tokens to Fuse to add new functionality and faster and cheaper verification times. You can start by selecting an initial sum, sigining the transaction and wait for 2 confirmations. Then you can switch to the Fuse chain to see the coins on the other side. Click here to learn more about the bridge.</div>
-              </ReactTooltip>
-              {!get(community, 'isMultiBridge', false) && <ToggleBridgeVersion isUseMultiBridge={get(community, 'isMultiBridge', false)} />}
+      {getCoverPhotoUri(communityMetadata) && (
+        <CoverPhoto
+          symbol={foreignToken && foreignToken.symbol}
+          metadata={communityMetadata}
+        />
+      )}
+      <div className='content__wrapper'>
+        <Header
+          withLogo={!getCoverPhotoUri(communityMetadata)}
+          metadata={communityMetadata}
+          tokenAddress={foreignToken && foreignToken.tokenAddress}
+          isClosed={community && community.isClosed}
+          communityURI={community && community.communityURI}
+          name={community && community.name}
+          networkType={networkType}
+          token={foreignToken}
+          handleJoinCommunity={userEntity ? undefined : handleJoinCommunity}
+        />
+        <CommunityInfo
+          tokensTotalSupplies={dashboard && dashboard.totalSupply}
+          foreignToken={foreignToken}
+          loadQrModal={loadQrModal}
+          communityAddress={community && community.communityAddress}
+          homeTokenAddress={homeTokenAddress}
+          foreignTokenAddress={community && community.foreignTokenAddress}
+        />
+        {
+          (community && community.foreignTokenAddress) && (
+            <div className='content__bridge'>
+              <div className='grid-x align-justify'>
+                <h3 className='content__bridge__title'>Bridge <FontAwesome style={{ fontSize: '60%' }} data-tip data-for='bridge' name='info-circle' /></h3>
+                <ReactTooltip className='tooltip__content' id='bridge' place='bottom' effect='solid'>
+                  <div>Use the bridge to move tokens to Fuse to add new functionality and faster and cheaper verification times. You can start by selecting an initial sum, sigining the transaction and wait for 2 confirmations. Then you can switch to the Fuse chain to see the coins on the other side. Click here to learn more about the bridge.</div>
```

Comment:
the text for V1 or V2? Cause for V2 the process is a bit different (you sign two txs)